### PR TITLE
Fix: fix signedKey using platform id

### PR DESCRIPTION
**Pull request by FogDong**
Signed-off-by: FogDong <dongtianxin.tx@alibaba-inc.com>

### Description of your changes

Fixes #

I have:

- [ ] Read and followed KubeVela's [contribution process](https://github.com/kubevela/kubevela/blob/master/contribute/create-pull-request.md).
- [ ] [Related Docs](https://github.com/kubevela/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary. 
- [ ] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### Special notes for your reviewer

@barnettZQG

### CODE REVIEW

File: cmd/apiserver/main.go [hunk 1]
```diff
@@ -52,6 +54,7 @@ func main() {
 	flag.BoolVar(&s.serverConfig.DisableStatisticCronJob, "disable-statistic-cronJob", false, "close the system statistic info calculating cronJob")
 	flag.Float64Var(&s.serverConfig.KubeQPS, "kube-api-qps", 100, "the qps for kube clients. Low qps may lead to low throughput. High qps may give stress to api-server.")
 	flag.IntVar(&s.serverConfig.KubeBurst, "kube-api-burst", 300, "the burst for kube clients. Recommend setting it qps*3.")
+	flag.StringVar(&service.SignedKey, "signed-key", utils.RandomString(16), "the signed key for token")
```
Comment (wonderflow) [inline cmd/apiserver/main.go:57]:
```suggestion
	flag.StringVar(&service.SignedKey, "signed-key", utils.RandomString(16), "the signed key for token, equals to the platform secret of velaux")
```

**Review by barnettZQG — APPROVED** [review]
LGTM

File: pkg/apiserver/domain/service/system_info.go [hunk 2]
```diff
@@ -63,6 +63,7 @@ func (u systemInfoServiceImpl) Get(ctx context.Context) (*model.SystemInfo, erro
 		}
 		return info, nil
 	}
+	info.SignedKey = rand.String(32)
```
Comment (wonderflow) [inline pkg/apiserver/domain/service/system_info.go:66]: will the random string be persistented ?

**Comment by wonderflow** [comment]
it still has potential risks if we store them as configmap(database).

File: pkg/apiserver/domain/service/authentication.go [hunk 3]
```diff
@@ -203,6 +204,7 @@ func (a *authenticationServiceImpl) generateJWTToken(username, grantType string,
 		},
 		Username:  username,
 		GrantType: grantType,
+		TimeStamp: time.Now(),
```
Comment (barnettZQG) [inline pkg/apiserver/domain/service/authentication.go:207]: Is this field unuseful? Remove it.

**Review by wonderflow — APPROVED** [review]

**Pull request merged**